.github/dependabot.yml

```diff
@@ -2,9 +2,22 @@
 
 version: 2
 updates:
-
+  # Check for updates to GitHub Actions every month
   - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:      
+      interval: "monthly"
+    labels:
+        - "github_actions"
```

Comment:
I think we should probably add this to the `omit-commits` plugins in `.autorc` so it is never mentioned in changelog etc. Looking at plugin source code. e.g.

```
[
  "omit-commits",
  {
    "labels": ["no-changelog", "github_actions"]
  }
],
```

Reply:
https://github.com/grafana/plugin-tools/pull/431/commits/8f4f3c33f8cc8c42e38b048cc22e94c9e77360cd